src/i18n/hi/nav.ts

```diff
@@ -1,11 +1,74 @@
 import { NavDictionary } from '../translation-checkers';
 
 export default NavDictionary({
-	// Start Here
 	startHere: 'यहाँ से शुरू करे',
 	'getting-started': 'पहले कदम',
+	install: 'स्थापित करें',
+	'editor-setup': 'संपादक सेटअप',
+	'guides/upgrade-to/v3': 'v3 में अपग्रेड करें',
 
-	// Core Concepts
-	coreConcepts: 'मूल अवधारणाएं',
-	'concepts/why-astro': 'क्यों Astro',
+	coreConcepts: 'मूल अवधारणाएँ',
+	'concepts/why-astro': 'Astro क्यों',
```

Comment:
This is admissible but with this, the [hi/why-astro](https://github.com/withastro/docs/blob/main/src/content/docs/hi/concepts/why-astro.mdx#L2) title tag should also be updated accordingly (nothing urgent but needs to addressed)